### PR TITLE
adds regulations, about us, newsletter pages placeholders and links i…n footer. Related to #489

### PR DESCRIPTION
**Pull request by tiriana**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/CC-5790
**Reference person:** @yourgithubname
**Description:**

Adds static page "O nas" - placeholder
Adds newsletter template and action
Changes links in footer in "Informacje" section

**New imports / dependencies:**

NONE

**Unit Tests:**
- `apps/volontulo/tests/views/test_newsletter.py`

**What tests do I need to run to validate this change:**

Check if links in footer lead to proper pages

### CODE REVIEW

File: apps/volontulo/templates/regulations.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{% block title %}Volontulo - podejmij pracę jako wolontariusz{% endblock %}
+
+{% block name %}newsletter-signup{% endblock %}
+
+{% block content-heading %}
+    <img class="img-responsive center-block" src="http://placehold.it/1260x260?text=Albo+jeden+duży+banner" />
+{% endblock %}
+
+{% block content %}
+    <h2>Zapisz się do newslettera</h2>
```
Comment (Yog) [inline apps/volontulo/templates/regulations.html]: the page filename is regulations.html, but you put here newsletter content. It should probably belong in newsletter_signup.html and vice versa

File: apps/volontulo/templates/newsletter_signup.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+{% block title %}Volontulo - podejmij pracę jako wolontariusz{% endblock %}
+
+{% block name %}newsletter-signup{% endblock %}
+
+{% block content-heading %}
+    <img class="img-responsive center-block" src="http://placehold.it/1260x260?text=Albo+jeden+duży+banner" />
+{% endblock %}
+
+{% block content %}
+    <h2>Regulamin</h2>
```
Comment (Yog) [inline apps/volontulo/templates/newsletter_signup.html]: you have messed up file names, look for comment below

**Comment by tomaszwozniak** [comment]
Please update the description

File: apps/volontulo/views/__init__.py [hunk 3]
```diff
@@ -263,6 +263,36 @@ def contact_form(request):
         }
     )
 
+def about_us(request):
```
Comment (tomaszwozniak) [inline apps/volontulo/views/__init__.py]: I believe that there is already logic related with rendering single pages

**Comment by tomaszwozniak** [comment]
Please fix build

**Comment by tiriana** [comment]
@magul I need help with 

`"pep8 --exclude='apps/volontulo/migrations/*,node_modules,.ropeproject' "`

it returns whole lot of errors from `./env/lib/python3.4/site-packages/faker/providers/address/pt_BR/__init__.py`

**Comment by tiriana** [comment]
@tomaszwozniak Description updated, build fixed. 

Please review.

**Comment by tomaszwozniak** [comment]
looks good

File: apps/volontulo/tests/views/test_newsletter.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+u"""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_newsletter.py:3]: Unneeded u"".

File: apps/volontulo/tests/views/test_newsletter.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+.. module:: test_newsletter
+"""
+from django.test import TestCase
+
+
+class TestNews(TestCase):
+    u"""Class responsible for testing newsletter specific views."""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_newsletter.py:10]: ditto

File: apps/volontulo/tests/views/test_newsletter.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+    # pylint: disable=invalid-name
+    def test__newsletter(self):
+        u"""Test getting newsletter signup page as anonymous."""
```
Comment (filipgorczynski) [inline apps/volontulo/tests/views/test_newsletter.py:14]: u"" again

File: apps/volontulo/views/__init__.py [hunk 7]
```diff
@@ -264,6 +264,17 @@ def contact_form(request):
     )
 
 
+def newsletter_signup(request):
+    u"""Newsletter signup page
```
Comment (filipgorczynski) [inline apps/volontulo/views/__init__.py:267]: u"" and no ful stop at the end of sentence.

**Comment by magul** [comment]
@tiriana You have merge conflict to resolve here.

**Pull request merged**